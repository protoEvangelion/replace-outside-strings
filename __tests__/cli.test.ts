import * as child_process from 'child_process'

test('check if cli works end to end', () => {
	const result = child_process.spawnSync(
		'npx',
		[
			'ts-node',
			'src/cli.ts',
			'-f',
			'__tests__/__fixtures__/index.ts',
			'--s1',
			'@include transition\\(',
			'--s2',
			'\\);',
			'--r1',
			'transition: ',
			'--r2',
			';',
		],
		{ cwd: process.cwd() }
	)

	expect(result.stdout.toString()).toMatchSnapshot()
})
