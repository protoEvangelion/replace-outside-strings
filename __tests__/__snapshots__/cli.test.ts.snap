// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check if cli works end to end 1`] = `
"export default \`
	.osb-nav-overlay {
		z-index: 0;

		transition: opacity 1s $navBezier;
		transition: yolo 3s ballin;
	}
\`
 ============================

"
`;
